srtp/srtp.c

```diff
@@ -3202,6 +3206,9 @@ static void srtp_calc_aead_iv_srtcp(srtp_session_keys_t *session_keys, v128_t *i
     in.v16[0] = 0;
     memcpy(&in.v16[1], &hdr->ssrc, 4); /* still in network order! */
     in.v16[3] = 0;
+    if (seq_num & 0x80000000) { /* the most significant bit should be zero */
```

Comment:
Maybe use a `#define MOST_SIG_BIT_32_BIT_WORD (0x80000000)` re #260 ?
(Terrible name made up on the fly :-) Not sure if it applies here though, since this is sort of a one-off value of which name would perhaps not make sense in other contexts such as 

~~~.c
crypto/math/datatypes.c=148=v128_bit_string(v128_t *x) {
crypto/math/datatypes.c-149-  int j, i;
crypto/math/datatypes.c-150-  uint32_t mask;
crypto/math/datatypes.c-151-  
crypto/math/datatypes.c-152-  for (j=i=0; j < 4; j++) {
crypto/math/datatypes.c:153:    for (mask=0x80000000; mask > 0; mask >>= 1) {
crypto/math/datatypes.c-154-      if (x->v32[j] & mask)
crypto/math/datatypes.c-155-    bit_string[i] = '1';
crypto/math/datatypes.c-156-      else
crypto/math/datatypes.c-157-    bit_string[i] = '0';
crypto/math/datatypes.c-158-      ++i;
crypto/math/datatypes.c-159-    }
crypto/math/datatypes.c-160-  }
crypto/math/datatypes.c-161-  bit_string[128] = 0; /* null terminate string */
crypto/math/datatypes.c-162-
crypto/math/datatypes.c-163-  return bit_string;
crypto/math/datatypes.c-164-}
~~~